simulator-archetypes/archetype-wsdl/src/main/resources/archetype-resources/pom.xml

```diff
@@ -11,6 +11,7 @@
   <packaging>war</packaging>
 
   <properties>
+    <java.version>11</java.version>
```

Comment:
Default should still be Java 8

Reply:
Done in 6bd41c1deb2a348d4f6dcee2f6e1f48c0376b6d3.